packages/share_plus/share_plus/example/lib/main.dart

```diff
@@ -208,15 +237,18 @@ class DemoAppState extends State<DemoApp> {
     final result = await Share.shareXFiles(
       [
         XFile.fromData(
-            buffer.asUint8List(data.offsetInBytes, data.lengthInBytes),
-            name: 'flutter_logo.png',
-            mimeType: 'image/png'),
+          buffer.asUint8List(data.offsetInBytes, data.lengthInBytes),
+          name: 'flutter_logo.png',
+          mimeType: 'image/png',
+        ),
       ],
       sharePositionOrigin: box!.localToGlobal(Offset.zero) & box.size,
     );
 
-    scaffoldMessenger.showSnackBar(SnackBar(
-      content: Text("Share result: ${result.status}"),
-    ));
+    scaffoldMessenger.showSnackBar(
+      SnackBar(
+        content: Text("Share result: ${result.status}"),
+      ),
+    );
```

Comment:
Only one possible improvement, it would be to refactor this showSnackBar with the one in line 217 into one method, so you can call both passing the result object.

Reply:
Updated